tools/data/add_transcripts_from_gff3_to_annotations.pl

```diff
@@ -142,6 +144,7 @@ sub print_usage {
     [--success_log|-l <success log file>]
     [--error_log|-L <error log file>]
     [--skip|-s <skip id file>]
+    [--test|-x]
```

Comment:
@deepakunni3  I think would change this to --verbose|-v   and make the description 'verbose output' or something like that.   Other than that looks perfect.